app/models/post.rb

```diff
@@ -0,0 +1,4 @@
+class Post < ApplicationRecord
+  belongs_to :user, class_name: 'User'
+  has_many :comments
```

Comment:
- [ ] It would be better if the Post model has a method that returns the `5 most recent comments for a given post`